functions/test-profile-details.js

```diff
@@ -65,6 +99,7 @@ exports.handler = async function fetchTestProfileDetails(event, context) {
 			// Even though the function implementation might change (making the response not immutable).
 			// Since this is a developer tool we can always advise to clear cache.
 			"Cache-Control": "immutable, max-age=86400",
+			"Content-Type": "application/json",
```

Comment:
Drive-by change which means that other software (e.g. postman) automatically applies sensible formatting.